src/main/gov/nasa/jpf/D.java

```diff
@@ -0,0 +1,8 @@
+package gov.nasa.jpf;
```

Comment:
These files should be in `src/test`.

Reply:
Please remove this file (as it is no longer needed, we have `src/test/.../D.java` now.